src/Commands/ExportPostmanCommand.php

```diff
@@ -211,14 +211,16 @@ protected function buildTree(array &$routes, array $segments, array $request): v
 
     public function makeRequest($route, $method, $routeHeaders, $requestRules)
     {
+        $routeStr = str_replace('}', '}}', str_replace('{', '{{', $route->uri()));
```

Comment:
Unless there is a specific reason for calling `str_replace` twice, here is a cleaner way to achieve the same thing.

```suggestion
        $routeStr = str_replace(['}', '{'], ['}}', '{{'], $route->uri());
```

Reply:
Yeah sounds good, just getting back into PHP after a few years away so I'm a little rusty. Thanks!